src/axom/quest/tests/quest_point_in_cell_mfem.cpp

```diff
@@ -107,16 +105,17 @@ template <int DIM, typename ExecSpace>
 class PointInCellTest : public ::testing::Test
 {
 public:
-  typedef axom::primal::BoundingBox<double, DIM> BBox;
-  typedef axom::primal::Point<double, DIM> SpacePt;
-  typedef axom::primal::Vector<double, DIM> SpaceVec;
+  using BBox = axom::primal::BoundingBox<double, DIM>;
+  using SpacePt = axom::primal::Point<double, DIM>;
+  using SpaceVec = axom::primal::Vector<double, DIM>;
   // _quest_pic_typedef_start
-  typedef axom::primal::Point<int, DIM> GridCell;
+  using GridCell = axom::primal::Point<int, DIM>;
```

Comment:
Should the int type here be the IndexType below?

Reply:
Thanks -- I made the change.
I also improved the consistency of the types for the `PointInCell` classes and worked through a few other issues.